package/src/index.ts

```diff
@@ -0,0 +1,537 @@
+                remaining: Number(req.response ? req.response.headers["x-ratelimit-remaining"] : req.headers["x-ratelimit-remaining"]),
+                reset: Number(req.response ? req.response.headers["x-ratelimit-reset"] : req.headers["x-ratelimit-reset"]),
+            },
+            error: req.response ? this._parseBody(req.response.data) : null,
+            url: req.config.url
+        }
+    }
+
+    /**
+	 * @internal
+	 * Make sure each key has a non empty value
+	 * @param input - Object to check for non empty values
+	 * @throws {@link TypeError} - Only if there is a empty value to a key
+	 */
+    private async _validateArgs(input: { [key: string]: any }) {
+        Object.keys(input).forEach(key => {
+            if (input[key] === null || input[key] === undefined) throw new TypeError(`Missing parameter: "${key}"`)
+        })
+    }
+
+    /**
+	 * Get an image of a valorant crosshair by its code
+	 * @example
+	 * Load and write a crosshair to crosshair.png
+	 * ```js
+	 * import { writeFileSync } from "fs"
+	 * 
+	 * const crosshair = await default.getCrosshair("0;s;1;P;c;5;o;1;d;1;z;3;0b;0;1b;0;S;c;4;o;1")
+	 * writeFileSync("crosshair.png", crosshair.data)
+	 * ```
+	 * @param code - Valorant crosshair code
+	 * @param size - (optional) Image size (default: `1024`)
+	 * @returns The image of the crosshair as a {@link Buffer}
+	 */
+    async getCrosshair(code: string, size?: number): Promise<APIResponse<CrosshairResponse>> {
+        this._validateArgs({ code })
+        return this._fetch<CrosshairResponse>("v1/crosshair/generate", { id: code, size }, {
+            responseType: "arraybuffer"
+        })
+    }
+
+    /**
+	 * Get all announcements from the valorant website of a country
+	 * @param countryCode - Country code of website
+	 * @returns List of announcements from the valorant website
+	 */
+    async getAnnouncements(countryCode: "en-us" | "en-gb" | "de-de" | "es-es" | "es-mx" | "fr-fr" | "it-it" | "ja-jp" | "ko-kr" | "pt-br" | "ru-ru" | "tr-tr" | "vi-vn"): Promise<APIResponse<WebsiteResponse>> {
+        this._validateArgs({ countryCode })
+        return this._fetch<WebsiteResponse>(`v1/website/${countryCode}`)
+    }
+
+    /**
+	 * Get information about valorant in a region, such as the client version, branch, and server version
+	 * @param region - Region of valorant to get info about
+	 * @returns Information about a regions valorant
+	 */
+    async getVersion(region: Region): Promise<APIResponse<VersionResponse>> {
+        this._validateArgs({ region })
+        return this._fetch<VersionResponse>(`v1/version/${region}`)
+    }
+
+    /**
+	 * Get all of the featured items in the current valorant store
+	 * @returns Featured items in the current valorant store
+	 */
+    async getFeaturedItems(): Promise<APIResponse<FeaturedItemsResponse>> {
+        // ! v2 is bugged for some reason
```

Comment:
tbd

Reply:
Is this a bug with the API or the way I'm calling it. I just have it commented here so in the future, when v2 does get fixed, I can change it